templates/templates/footer.html

```diff
@@ -10,7 +10,7 @@
             <a class="p-link--soft" href="/about">关于我们</a>
           </li>
           <li class="p-inline-list__item">
-            <a class="p-link--soft" href="/contact" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact Us', 'eventLabel' : 'cn.ubuntu.com - Footer link', 'eventValue' : undefined });">联系我们</a>
+            <a class="p-button--neutral js-invoke-modal" href="/contact" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact Us', 'eventLabel' : 'cn.ubuntu.com - Footer link', 'eventValue' : undefined });">联系我们</a>
```

Comment:
Do you use a modal for this button? For me it takes me straight to the contact page instead of showing a modal. If no modal used, please remove `js-invoke-modal` class.

Reply:
Removed